v2/sourcedb-to-spanner/terraform/samples/sharded-bulk-migration/README.md

```diff
@@ -0,0 +1,431 @@
+## Sample Scenario: Sharded MySQL to Spanner production bulk migration
```

Comment:
How is this different from multiple sources ?

Reply:
multiple sources runs any random n sources, like mentioned above might make sense to remove it since its confusing and not a strong use case for terraform.